contracts/eosiolib/eosiolib.cpp

```diff
@@ -202,7 +202,6 @@ namespace eosio {
          {
             _heap_size = size;
             _heap = mem_heap;
-            memset(_heap, 0, _heap_size);
```

Comment:
`_heap` declared as `char*` in  method: `memory* next_active_heap()` under class`memory_manager` in memory.hpp together with uint32_t `_heap_size` and `_offset`